### ISSUE TITLE
Support deployment with lambda@edge

### ISSUE THREAD

**Issue by jlarmstrongiv** [sst/sst#4270]
The SsrSite Construct in v2 supported lambda@edge deployment for Astro, Next.js, Remix, SvelteKit, and more.  

I deploy Astro with lambda@edge for 75% of my websites in SST v2.  

Supporting deployment with lamdba@edge is my main blocker to upgrading to v3.

- [x] I use the Team plan on seed.run to deploy

**Comment by fwang** [comment]
Starting v3.10, `SsrSite` moved away from using L@E. Instead, you can specify the [`regions`](https://sst.dev/docs/component/aws/nextjs/#regions) prop to deploy the server function into multiple regions.

This is a part of a bigger change to `SsrSite`. You can read more here https://github.com/sst/sst/releases/tag/v3.10.0

Closing this for now.

**Comment by jlarmstrongiv** [comment]
@fwang does the `regions` prop only apply to `SsrSite`?  In my routers, I have regular lambdas, s3 buckets, and urls where I would like to apply regional routing as well.

**Comment by fwang** [comment]
yeah currently just `SsrSite`. But if you have a specific use case, feel free to open a GH issue. I think we can add multi-region support to the `Router` component. And route to different functions/buckets depending on user's region.

**Comment by jlarmstrongiv** [comment]
Gotcha, I used vanilla lambda@edge functions with s3 buckets in multiple regions, so I’m still unable to replicate what I had in sst v2.

**Comment by jlarmstrongiv** [comment]
I created https://github.com/sst/sst/issues/5800 as a followup issue to add region support to lambda functions, s3 buckets, and urls.
